Landing-page

Chanisha opened the pull request:
logo in the top
text for the landing-page
image in the footer
added the sign-in google button
made the new page for adding tasks
made the sidebar
added the deatails like home, user,etc
made the mainpage for the tasks area

Chanisha committed:
develop landing-page
logo in the top
text for the landing-page
image in the footer

Please enter the commit message for your changes. Lines starting

Chanisha committed:
made new page
added the sign-in google button
made the new page for adding tasks
made the sidebar
added the deatails like home, user,etc
made the mainpage for the tasks area

csachdeva83 commented on package-lock.json:
Add package-lock.json in .gitignore file. From subsequent commits

csachdeva83 commented on public/logo.png:
For product logo images download SVG file from figma. Don't use png for logo.

SVG images dimensions are flexible, pixels on stretch don't deteriorate

csachdeva83 commented on src/app/globals.css:
```diff
@@ -1,26 +1,122 @@
-  background: var(--background);
-  color: var(--foreground);
-  font-family: Arial, Helvetica, sans-serif;
+.dark {
```
Are we creating website for dark mode as well ?
If not then remove dark class.

Because unnecessary css increases bundle size.

csachdeva83 commented on src/app/google-auth/page.tsx:
Rename google-auth to auth.

URL should general purposed. In URL don't mention what specific in present in it.

Make it /auth

csachdeva83 commented on src/app/google-auth/page.tsx:
```diff
@@ -0,0 +1,81 @@
+import { Avatar, AvatarFallback } from "@/components/ui/avatar";
+
+
+export default function GoogleAuthPage() {
```
Make the function name Auth()

csachdeva83 commented on src/app/google-auth/page.tsx:
```diff
@@ -0,0 +1,81 @@
+  label: string;
+  isActive?: boolean;
+}) {
+  const classes = `w-full justify-start gap-2 rounded-md ${
```
Use cn from utils.ts

csachdeva83 commented on src/app/google-auth/page.tsx:
```diff
@@ -0,0 +1,81 @@
+
+  return (
+    <Button
+      variant={isActive ? "secondary" : "ghost"}
```
Use cn from utils.ts

csachdeva83 commented:
Make the pages responsive

Chanisha committed:
made some important changes
put the packagelock in gitignore
removed the dark class
changed google-auth to auth

Chanisha committed:
g

Chanisha committed:
made a new page
made a modal dialog box
changed the design on home while is selected
gave a border colour on cancel

csachdeva83 commented on public/avatar.png:
svg only

csachdeva83 commented on public/house.png:
svg only

Chanisha replied:
I have changed the file format

csachdeva83 commented on public/illustration.png:
svg only

csachdeva83 commented on public/logo.png:
svg only

csachdeva83 commented on public/notepencil.png:
svg only

csachdeva83 commented on public/usercircle.png:
svg only

csachdeva83 commented on src/app/auth/page.tsx:
```diff
@@ -0,0 +1,148 @@
+              Create New Board
+            </Button>
+          </div>
+        </main>
```
Make main only div

csachdeva83 commented on src/app/auth/page.tsx:
```diff
@@ -0,0 +1,148 @@
+      {isModalOpen && (
+        <div className="fixed inset-0 z-50 flex items-center justify-center bg-black/70">
+        <div className="bg-white rounded-md shadow-lg p-6 w-full max-w-md">
+        <h2 className="text-lg font-semibold mb-4">Create new board</h2>
```
Instead of heading tag like <h1>, <h2>, etc.

Use <span> or <p>
Use css like text-size, font-weight. To replicate heading tags

Chanisha committed:
change the file formats for images
change the file format for avatar, notepencil,ucercircle, etc as .svg
